plugins/unmanaged_files/unmanaged_files_inspector.rb

```diff
@@ -69,28 +83,18 @@ def extract_rpm_database
 
       # make sure that dirs leading to a managed file are treated as if they were
       # in rpm database, otherwise we cannot exclude whole unmanaged trees
-      while( !dirh.has_key?(dir) && dir.size > 1 )
-        dirh[dir] = false
+      while( !dirs.has_key?(dir) && dir.size > 1 )
```

Comment:
`Hash#has_key?` is deprecated in favor of `Hash#key?`.<br>Use space after control keywords.<br>Space inside parentheses detected.